### PR TITLE
adding myself

### PR DESCRIPTION
**Pull request by ioana-chiorean**

### CODE REVIEW

**Comment by jancborchardt** [comment]
@ioana-chiorean welcome! I guess there’s a typo in »Cuj-Napoca« ;)

**Comment by ioana-chiorean** [comment]
Indeed. Changed it .

**Comment by elioqoshi** [comment]
Welcome Ioana!

**Comment by jancborchardt** [comment]
(@elioqoshi you can delete branches after they were merged ;)

File: people/ioana-chiorean [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: QA Team Lead, Open Source Evanghelist, Open Designer Enthusiast
```
Comment (Volker-E) [inline people/ioana-chiorean]: It could be a try to mislead search engines, but evangelist is written wrong -- both times.

Comment (ioana-chiorean) [inline people/ioana-chiorean]: Thanks @Volker-E -trying this again https://github.com/opensourcedesign/organization/pull/25

**Pull request merged**